### PR TITLE
Ru: any quest and much more

### PR DESCRIPTION
**Pull request by Sinitsaa**

### CODE REVIEW

File: translated files/Quests/base protection/.deleteme [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (alama) [inline translated files/Quests/base protection/.deleteme:1]: For what?

File: translated files/Quests/Luzer/.deleteme [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (alama) [inline translated files/Quests/Luzer/.deleteme:1]: For what?

File: translated files/Quests/Elder/.deleteme [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (alama) [inline translated files/Quests/Elder/.deleteme:1]: For what?

File: translated files/Quests/base protection/.deleteme (hunk 1 above)
Comment (Sinitsaa) [reply translated files/Quests/base protection/.deleteme:1]: Classification of emergency quests.
A little later I will remove it.

File: translated files/Quests/Luzer/.deleteme (hunk 2 above)
Comment (Sinitsaa) [reply translated files/Quests/Luzer/.deleteme:1]: Classification of emergency quests.
A little later I will remove it.

File: translated files/Quests/Elder/.deleteme (hunk 3 above)
Comment (Sinitsaa) [reply translated files/Quests/Elder/.deleteme:1]: Classification of emergency quests.
A little later I will remove it.

File: translated files/UI/ui_pet.csv [hunk 4]
```diff
@@ -130,7 +130,7 @@ DirectBuy_EggSynthesisNum_FromButton_Msg#0,"""<pict StarGem><c 1dedd7>SG消費
 DirectBuy_EggSynthesisNum_FromEmpty_Title#0,"""Подтверждение покупки предмета"""
 DirectBuy_EggSynthesisNum_FromEmpty_Msg#0,"""<pict StarGem><c 1dedd7>SG消費コンテンツ<c><br><br>$(0)の<br>合成可能数が不足しています。<br><br>「合成可能数回復」の購入確認画面に<br>進みますか？"""
 DirectBuy_EggSynthesisNum_Confirm_Title#0,"""Подтверждение покупки предмета"""
-DirectBuy_EggSynthesisNum_Confirm_Msg#0,"""以下のアイテムを$(0)SG(通常100SG)で購入し、<br>合成可能数を５個回復させます。<br>Are you sure?<br><yellow><br>※連続購入時、購入１回目～９回目までは<br>\u3000価格が割引されますが、回数が増えると<br>\u3000割引率が下がります。<c><br>※Number of purchases resets at 5am every day.<br>(Current SG: $(1))"""
+DirectBuy_EggSynthesisNum_Confirm_Msg#0,"""以下のアイテムを$(0)SG(通常100SG)で購入し、<br>合成可能数を５個回復させます。<br>Are you sure?<br><yellow><br>※連続購入時、購入１回目～９回目までは<br>\u3000価格が割引されますが、回数が増えると<br>\u3000割引率が下がります。<c><br>※Number of purchases resets at 5am every day.<br>(Current SG: $(1))" =~ "以下のアイテムを$(0)SG（通常100SG）で購入し、<br>合成可能数を５個回復させます。<br>よろしいですか？<br><yellow><br>※連続購入時、購入１回目～９回目までは<br>\u3000価格が割引されますが、回数が増えると<br>\u3000割引率が下がります。<c><br>※購入回数は毎日朝5時にリセットされます。<br>（所持SG:$(1))"""
```
Comment (alama) [inline translated files/UI/ui_pet.csv]: Really? are you just copying and paste from the check script output

File: translated files/UI/ui_pet.csv [hunk 5]
```diff
@@ -143,7 +143,7 @@ DirectBuy_EggSynthesisNum_Item08#0,"""Восстановление заряда
 DirectBuy_EggSynthesisNum_Item09#0,"""Восстановление заряда синт. (9-й)"""
 DirectBuy_EggSynthesisNum_Item10#0,"""Восстановление заряда синт. (10-й+)"""
 DirectBuy_EggSynthesisNum_Success_Title#0,"""Предмет успешно приобретён"""
-DirectBuy_EggSynthesisNum_Success_Msg#0,"""$(0)の<br>合成可能数を５つ追加しました。<br><br><yellow>※この５回分は、エッグ合成時の<br>\u3000獲得経験値量にボーナスがつきます。<c><br><br>※Number of purchases resets at 5am every day."""
+DirectBuy_EggSynthesisNum_Success_Msg#0,"""$(0)の<br>合成可能数を５つ追加しました。<br><br><yellow>※この５回分は、エッグ合成時の<br>\u3000獲得経験値量にボーナスがつきます。<c><br><br>※Number of purchases resets at 5am every day." =~ "$(0)の<br>合成可能数を５つ追加しました。<br><br><yellow>※この５回分は、エッグ合成時の<br>\u3000獲得経験値量にボーナスがつきます。<c><br><br>※購入回数は毎日朝5時にリセットされます。"""
```
Comment (alama) [inline translated files/UI/ui_pet.csv]: What the fuck are you doing?

**Review by alama — CHANGES_REQUESTED** [review]

File: translated files/UI/ui_pet.csv (hunk 5 above)
Comment (Sinitsaa) [reply translated files/UI/ui_pet.csv]: I'm trying to correct mistakes. Sometimes I accidentally create new ones.

**Comment by Sinitsaa** [comment]
Merge, please

**Pull request merged**